service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AisConsentCreationSteps.java

```diff
@@ -290,35 +298,62 @@ public void psuTriesToAuthoriseConsent(String psuId, String password) {
         StartScaprocessResponse.class);
 
     String authorisationId = TestUtils
-        .extractAuthorisationId((String) startScaResponse.getBody().getLinks()
-            .get("startAuthorisationWithPsuAuthentication"));
+        .extractId((String) startScaResponse.getBody().getLinks()
+            .get("startAuthorisationWithPsuAuthentication"), "authorisations");
 
-    PsuData psuData = new PsuData();
-    psuData.setPassword(password);
+    String url = "consents/" + context.getConsentId() + "/authorisations/" + authorisationId;
```

Comment:
`String.format()` or `UriComponentsBuilder` would be an option. Or `Joiner.on("/").join("consents", context.getConsentId, "authorisations", authorisationId)`.

Reply:
Done